### ISSUE TITLE
Failed to update to 1.12.3 on Hoobs

### ISSUE THREAD

**Issue by jchristianj** [werthdavid/homebridge-weather#34]
Hi,
Hoobs is indicating that an update is available, but upon installation the log shows:

...
17.6.2020, 12:58:27 - npm
17.6.2020, 12:58:27 -  ERR! code EEXIST
npm ERR! path /home/hoobs/.hoobs/node_modules/gtoken/node_modules/.bin/mime
17.6.2020, 12:58:27 - npm
17.6.2020, 12:58:27 - ERR! Refusing to delete /home/hoobs/.hoobs/node_modules/gtoken/node_modules/.bin/mime: is outside /home/hoobs/.hoobs/node_modules/gtoken/node_modules/mime and not a link
npm ERR! File exists: /home/hoobs/.hoobs/node_modules/gtoken/node_modules/.bin/mime
npm ERR! Remove the existing file and try again, or run npm
npm ERR! with --force to overwrite files recklessly.
17.6.2020, 12:58:27 - 
17.6.2020, 12:58:27 - npm ERR! A complete log of this run can be found in:
npm ERR!     /home/hoobs/.npm/_logs/2020-06-17T10_58_27_717Z-debug.log
17.6.2020, 12:58:27 - Plugin "homebridge-weather" updated.
17.6.2020, 12:58:27 - Got SIGINT, shutting down Bridge...
...

it doesn't update... so what can I do?

Thanks,
Christian

**Comment by werthdavid** [comment]
I don't have a Hoobs installation but I'll see what I can do

**Comment by jchristianj** [comment]
Thanks a lot. If I can support you with logs or other stuff let me know.

**Comment by werthdavid** [comment]
Are you sure this comes from the homebridge-weather plugin?

**Comment by jchristianj** [comment]
Think so. It only happens when I try to update the weather plugin. 
Maybe it happens in hoobs because of a different plugin handling? If necessary I can check with the hoobs support people on Facebook or so.

**Comment by werthdavid** [comment]
That would be great. I am right that you can download Hoobs only if you pay?

**Comment by jchristianj** [comment]
No it's free: https://github.com/hoobs-org/HOOBS/releases
3.2.6 is the version I have installed. Just flash an SD card with it (use Balena Etcher tool), put it in your Pi to set it up and then you can install homebridge plugins.

**Comment by jchristianj** [comment]
Posted for help on Facebook, but only recommendation was to use another (hoobs certified) weather plugin, which I refused, because I like the simplicity of yours and don't want to re-arrange my home automations.
Without knowing what I was doing I desperately connected by terminal to hoobs, googled some commands, navigated to the folder where the non-deletable file has been located, and removed it with rm --force command.
Then I pushed the update-button for your plugin in hoobs... et voila, it updated! Hoping the next time it will not be so thrilling... cheers.
